File: src/main/java/no/nav/pensjon/regler/domain/beregning2011/LonnsvekstInformasjon.java
package no.nav.pensjon.regler.domain.beregning2011;



import java.io.Serializable;
import java.util.Date;

public class LonnsvekstInformasjon implements Serializable {
    private static final long serialVersionUID = -979400824825729782L;


    private double lonnsvekst;


    private Date reguleringsDato;


    private int uttaksgradVedRegulering;

    public LonnsvekstInformasjon() {
        super();
    }

    public LonnsvekstInformasjon(LonnsvekstInformasjon li) {
        this();
        lonnsvekst = li.lonnsvekst;
        if (li.reguleringsDato != null) {
            reguleringsDato = li.reguleringsDato;
        }
        uttaksgradVedRegulering = li.uttaksgradVedRegulering;
    }

    /**
     * @return the lonnsvekst
     */
    public double getLonnsvekst() {
        return lonnsvekst;
    }

    /**
     * @param lonnsvekst the lonnsvekst to set
     */
    public void setLonnsvekst(double lonnsvekst) {
        this.lonnsvekst = lonnsvekst;
    }

    /**
     * @return the reguleringsDato
     */
    public Date getReguleringsDato() {
        return reguleringsDato;
    }

    /**
     * @param reguleringsDato the reguleringsDato to set
     */
    public void setReguleringsDato(Date reguleringsDato) {
        this.reguleringsDato = reguleringsDato;
    }

    /**
     * @return the uttaksgradVedRegulering
     */
    public int getUttaksgradVedRegulering() {
        return uttaksgradVedRegulering;
    }

    /**
     * @param uttaksgradVedRegulering the uttaksgradVedRegulering to set
     */
    public void setUttaksgradVedRegulering(int uttaksgradVedRegulering) {
        this.uttaksgradVedRegulering = uttaksgradVedRegulering;
    }

}

File: src/main/kotlin/no/nav/pensjon/regler/domain/beregning2011/LonnsvekstInformasjon.kt
package no.nav.pensjon.regler.domain.beregning2011

import java.io.Serializable
import java.util.*
import java.time.LocalDate

class LonnsvekstInformasjon : Serializable {
    /**
     * @param lonnsvekst the lonnsvekst to set
     */
    var lonnsvekst = 0.0

    /**
     * @param reguleringsDato the reguleringsDato to set
     */
    @Deprecated("Use reguleringsDatoLd instead")
    var reguleringsDato: Date? = null
    var reguleringsDatoLd: LocalDate? = null

    /**
     * @param uttaksgradVedRegulering the uttaksgradVedRegulering to set
     */
    var uttaksgradVedRegulering = 0
}